test/test-ghe-detect-leaked-ssh-keys.sh

```diff
@@ -63,7 +63,7 @@ begin_test "ghe-detect-leaked-ssh-keys leaked keys in old snapshot"
   echo 98:d8:99:d3:be:c0:55:05:db:b0:53:2f:1f:ad:b3:60 >> "$SHARED_UTILS_PATH/ghe-ssh-leaked-host-keys-list.txt"
 
   output=$(ghe-detect-leaked-ssh-keys -s "$GHE_DATA_DIR/2")
-  ! echo $ouput | grep -q "Leaked key in current backup"
+  ! echo $output | grep -q "Leaked key in current backup"
```

Comment:
as @lildude notes, this is a nontrivial bug fix. w/o this, the test is useless....